episodes/12-homework.md

```diff
@@ -41,7 +41,6 @@ To prepare for our next session, please:
   - Data Carpentry   
     - [Exploring Data with OpenRefine](https://datacarpentry.org/OpenRefine-ecology-lesson/03-exploring-data.html)
     - [Basic Queries in SQL](https://datacarpentry.org/sql-ecology-lesson/01-sql-basic-queries.html)
-    - [Starting with Data in R](https://datacarpentry.org/R-ecology-lesson/02-starting-with-data.html)
```

Comment:
I expect we will pretty quickly get requests for an R option here...

Reply:
Resolved in ce99df1

Reply:
This looks fine, although I feel like the Manipulating data episode is less likely than the ggplot2 one to result in people no getting to any output in their 3 minute practice teaching.